src/store/selectors/activities/index.ts

```diff
@@ -5,14 +5,14 @@ import {
   CreateMemberOperation,
   VerifyOperation,
   FlagMemberOperation,
-  ResolveFlagMemberOperation
+  ResolveFlagMemberOperation,
+  MintReferralBonusOperation,
+  MintBasicIncomeOperation
 } from "@raha/api-shared/dist/models/Operation";
 
 import {
   Activity,
   NewMemberRelatedOperations,
-  MintBasicIncomeOperation,
-  MintReferralBonusOperation,
   ActivityType,
   IndependentOperation
 } from "./types";
```

Comment:
Can we delete `MintBasicIncomeOperation`, `MintReferralBonusOperation`, and others from `.types`?